package/cfg/k3s-cis-1.6-hardened/config.yaml

```diff
@@ -0,0 +1,46 @@
+---
+## Version-specific settings that override the values in cfg/config.yaml
+
+master:
+  components:
+    - apiserver
+    - scheduler
+    - controllermanager
+    - etcd
+    - policies
+
+  apiserver:
+    bins:
+      - containerd
```

Comment:
clarification on using containerd binary: there is no way to use either k3s or k3s-server in the binary here, since the cmd for the k3s process is `k3s-server` (/proc/<k3s-pid>/cmd) and kubebench check for this cmd and compare it with the process name which is `/usr/local/bin/k3s server` so the safest bet is to use containred instead